src/main/java/javaslang/control/ValidationBuilder.java

```diff
@@ -0,0 +1,75 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _ _____
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  / /  _  \   Javaslang
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/  \__/  /   Copyright 2014-now Daniel Dietrich
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/___\_/  \_//  \__/_____/    Licensed under the Apache License, Version 2.0
+ */
+package javaslang.control;
+
+import javaslang.Function2;
+import javaslang.Function3;
+import javaslang.Function4;
+import javaslang.collection.List;
+
+public class ValidationBuilder<E,T1,T2> {
```

Comment:
Does it makes sense to put ValidationBuilder into a `ValidationModule` interface inside of `Validation`, like is done in `Match` and other places?